fakestorage/object.go

```diff
@@ -562,6 +562,8 @@ func (s *Server) downloadObject(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Length", strconv.Itoa(len(content)))
 	w.Header().Set(contentTypeHeader, obj.ContentType)
 	w.Header().Set("X-Goog-Generation", strconv.FormatInt(obj.Generation, 10))
+	w.Header().Set("X-Goog-Hash", fmt.Sprintf("crc32c=%s", obj.Crc32c))
```

Comment:
I'm not sure whether `obj.Crc32c` is generated by the server or comes from user input, so please check whether this value needs to be sanitized for use in the response header.

Reply:
Sorry I missed this comment originally, but yeah those are generated on upload and when seeding the data, so we should be good!